test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4318,6 +4425,41 @@ def cleanup_files(self):
             self.du.rm(path=dyn_files, sudo=True, force=True)
         self.dyn_created_files = []
 
+    def _get_log_type(self, hostname=None):
+        """
+        This function will return which log files to read
+
+        :param hostname: name of host from where to get log messages
+        :return: int or None
+        """
+        if hostname is None:
+            hostname = socket.gethostname()
+        pbs_locallog = self.du.parse_pbs_config(
+                       hostname=hostname).get("PBS_LOCALLOG")
+        pbs_syslog = self.du.parse_pbs_config(
+                     hostname=hostname).get("PBS_SYSLOG")
+
+        if pbs_syslog is None:
+            pbs_syslog = 0
+
+        if pbs_locallog is None:
+            pbs_locallog = 1
+
+        # file_to_check=1 for syslog, file_to_check=2 for local,
+        # file_to_check=3 for both
+
+        if int(pbs_syslog) == 0 and int(pbs_locallog) == 0:
+            raise ValueError('logging should be present in atleast one file')
```

Comment:
I would suggest raising some PBS specific exception from this place like PtlLogMatchError.
By definition ValueError are raised when the argument passed to a function has incorrect value, which isn't the case here - 
Here is the definition I found - `Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.`